pkg/tagger/collectors/kubelet_extract.go

```diff
@@ -99,6 +100,25 @@ func (c *KubeletCollector) parsePods(pods []*kubelet.Pod) ([]*TagInfo, error) {
 		// container tags
 		for _, container := range pod.Status.Containers {
 			lowC := append(low, fmt.Sprintf("kube_container_name:%s", container.Name))
+			// check image tag in spec
+			for _, containerSpec := range pod.Spec.Containers {
+				if containerSpec.Name == container.Name {
+					imageName, shortImage, imageTag, err := docker.SplitImageName(containerSpec.Image)
+					if err != nil {
+						log.Debugf("Error splitting %s: %s", containerSpec.Image, err)
```

Comment:
We should avoid the keyword `Error` and instead use words like `Cannot`. Especially when the severity is different than error.